nextflow.config

```diff
@@ -29,7 +29,8 @@ params {
     mode = 'single'
     tpm = 1
     fastp_additional_params = '-5 -3 -W 4 -M 20 -l 15 -x -n 5 -z 6'
-    histat2_additional_params = ''
+    hisat2_additional_params = ''
+    featurecounts_additional_params = ''
```

Comment:
do we want to put the default parameters here? Because then they would also occur in the help and give a better idea of how to use this parameter. Suggestion:

```
featurecounts_additional_params = '-t gene -g gene_id'
```

Reply:
I like that!